### DISCUSSION TITLE
Nuclei won't update

### DISCUSSION THREAD

**Discussion by Rainbow-Origins** [projectdiscovery/nuclei#3325] [Q&A] [answered]
Hello, I was having an issue with nuclei not updating. I run the command "nuclei -update", but it just keeps telling me "flag provided but not defined: -update" I looked it up and tried running "nuclei -un" as well since that seems to be the newer method, but it's the same thing. I am stuck with an outdated version. I don't know what the issue is. My version is stuck at 2.8.8 and I am in kali linux.

**Comment by ehsandeep** [comment, marked as answer]
@Rainbow-Origins see https://github.com/projectdiscovery/nuclei/discussions/2151

**Comment by olearycrew** [comment]
Closing discussion as it was tracked in #2151
